.aws/src/event-rules/account-delete-monitor/config.ts

```diff
@@ -7,5 +7,13 @@ export const config = {
     schema: 'queue-check-delete',
     bus: 'default',
   },
+  userMerge: {
+    name: 'user-merge',
+    schema: 'user-merge',
+    //defined in web repo under UserMergeEvent class
+    source: 'user-merge',
+    detailType: ['web-repo'],
```

Comment:
i'm a little unsure about the intention of `source` and `detailType`, but just based on the names, it seems these would be better the other way around? like `source: 'web-repo'` and `detailType: ['user-merge']`. ignore me if i am lacking context 😄

Reply:
yes you are right..but initially i set source as `user-merge` and now i am setting it like this to replay events 🤦‍♀️ 

 so keeping it this way..after replaying events, will swap it to correct source in both web repo, ADM and this repo..carving this out as a seperate ticket here: https://getpocket.atlassian.net/browse/INFRA-770